Firestore/Source/Public/FirebaseFirestore/FIRVectorValue.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(VectorValue)
+@interface FIRVectorValue : NSObject
+
+/** :nodoc: */
+- (instancetype)init NS_UNAVAILABLE;
+
+// Public initializer is required to support Codable
+- (instancetype)initWithNSNumbers:(NSArray<NSNumber *> *)data NS_REFINED_FOR_SWIFT;
```

Comment:
nit: `NS` prefix can be dropped in names and the initializer should match up with the corresponding property

```suggestion
// Public initializer is required to support Codable
- (instancetype)initWithArray:(NSArray<NSNumber *> *)array NS_REFINED_FOR_SWIFT;
```

Reply:
Implemented